cms-common/src/main/java/io/commercetools/sunrise/cms/CmsService.java

```diff
@@ -11,21 +11,12 @@
 @FunctionalInterface
 public interface CmsService {
 
-     /**
-     * Gets the content corresponding to the given CMS identifier for the first found given language.
-     * @param locales the list of locales used to translate the message
-     * @param cmsIdentifier identifier of the CMS entry field
-     * @return the {@code completionStage} of the content in the first found given language, or absent if it could not be found
-     */
-    CompletionStage<Optional<String>> get(final List<Locale> locales, final CmsIdentifier cmsIdentifier);
-
     /**
-     * Gets the content corresponding to the given CMS identifier for the first found given language.
-     * @param locales the list of locales used to translate the message
-     * @param cmsIdentifier identifier of the CMS content
-     * @return the {@code completionStage} of the content in the first found given language, or empty string if it could not be found
+     * Gets the page content corresponding to the given key.
+     * @param pageKey identifying the page
```

Comment:
Yes, I've done that in my project and works well. I hope it's not too complicated for a merchant, though. I'm still trying to find out more about it, but so far we have an option there.